vorestation.dme

```diff
@@ -2425,6 +2425,7 @@
 #include "code\modules\lighting\lighting_area.dm"
 #include "code\modules\lighting\lighting_atom.dm"
 #include "code\modules\lighting\lighting_corner.dm"
+#include "code\modules\lighting\lighting_fake_sun.dm"

```

Comment:
This should probably have _vr appended to the end so I know it is a Vorestation specific file.

Reply:
So piiiiicky